CIPs/cip-5.md

```diff
@@ -3,7 +3,7 @@ cip: 5
 title: Streamtypes
 author: Michael Sena (@michaelsena), Joel Thorstensson (@oed), Janko Simonovic
 discussions-to: https://github.com/ceramicnetwork/CIP/issues/37
-status: Review
+status: Final
```

Comment:
similar to my comment about the anchor contract CIP - there is still a lot of language in this CIP that uses "document" instead of "stream", and "Record" instead of "Commit".  Do we need to update the terminology to be consistent with what we use elsewhere before finalizing this?